macros/edr/materializations/model/incremental.sql

```diff
@@ -20,8 +29,8 @@
   {% do return(relations) %}
 {% endmaterialization %}
 
-{% materialization incremental, adapter="bigquery" %}
-  {% set relations = dbt.materialization_incremental_bigquery() %}
+{% materialization incremental, adapter="bigquery", supported_languages=["sql", "python"] %}
```

Comment:
Currently testing if this would break `dbt<1.3.0` users because `supported_languages` was unknown.

Reply:
It does break it.